src/components/Nav.js

```diff
@@ -0,0 +1,18 @@
+import { NavLink } from 'react-router-dom';
+import { useToken } from '../context/TokenContext';
+
+export default function Nav() {
+  const { hasToken } = useToken();
+
+  return (
+    <nav>
+      {' '}
+      {hasToken && (
+        <>
+          <NavLink to="/list">Item List</NavLink>
+          <NavLink to="/add-item">Add Item</NavLink>
+        </>
+      )}
+    </nav>
+  );
+}
```

Comment:
After completing the user story from the first week, I realized it would have been a good idea to separate the nav section into its own component. Nicely done here.